resources/assets/coffee/forum/topic-ajax.coffee

```diff
@@ -70,17 +75,31 @@ $(document).on 'ajax:success', '#forum-topic-reply-box', (_event, data) ->
     newPost = window.forum.endPost()
     newPost.classList.remove('js-forum-post__shrunk')
     newPost.scrollIntoView()
+    window.isDoublePost = true
+
+    $('#forum-topic-reply-box').hide()
+
+    doublepostBox = $("#doublepost-box")
+    editLastPostLink = doublepostBox.find("a")
+    lastPostId = $(".forum-post").last().attr("data-post-id")
```

Comment:
at this point might as well have the double post box target as `lastPost` instead of specifying id. Or a different link type altogether (`edit-last-post` link or something like that which clicks last post's edit link).